app/SmartAlbums/BaseSmartAlbum.php

```diff
@@ -40,9 +40,9 @@ abstract class BaseSmartAlbum implements AbstractAlbum
 	protected PhotoQueryPolicy $photoQueryPolicy;
 	protected string $id;
 	protected string $title;
-	protected bool $isPublic;
-	protected bool $isDownloadable;
-	protected bool $isShareButtonVisible;
+	protected bool $grant_download;
+	protected bool $is_public;
+	protected bool $is_share_button_visible;
```

Comment:
I am not 100% sure anymore, but I thought we said that we wanted to make this a global option only (also see previous remarks).